docs/devices.rst

```diff
@@ -0,0 +1,147 @@
+Devices
+=======
+
+*Devices*, also known as *endpoints*, are at the heart of Carbon Black Cloud's functionality.  Each device has a
+Carbon Black Cloud sensor installed on it, which communicates with Carbon Black analytics and the Carbon Black Cloud
+console.
```

Comment:
The devices do not communicate with the Carbon Black Cloud console. I'd simplify this to Carbon Black Cloud backend

Reply:
Once more, I just phrased it the way the user guide did.